.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,30 @@
+<!---
+Thank you for contributing to NVIDIA DALI.
+
+Please fill the relevant information in this PR template.
+
+Fields with `- [ ]` represent check-boxes that can be marked after you create and save the Pull Request.
+--->
+
 ## Description
 - [ ] **Bug fix** (*non-breaking change which fixes an issue*)
 - [ ] **New feature** (*non-breaking change which adds functionality*)
 - [ ] **Breaking change** (*fix or feature that would cause existing functionality to not work as expected*)
 - [ ] **Refactoring** (*Redesign of existing code that doesn't affect functionality*)
 - [ ] **Other** (*e.g. Documentation, Tests, Configuration*)
 
-### Describe in detail what happened in this PR
+### What happened in this PR
```

Comment:
```suggestion
### What happened in this PR?
```

Reply:
I prefer to leave out the punctuation here.